README.md

```diff
@@ -11,7 +11,7 @@ In config.js:
 ``` javascript
 System.config({
   map: {
-    "googlmaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!HeinrichFilter/systemjs-plugin-googlemaps/googlemaps"
+    "googlemaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!systemjs-googlemaps"
```

Comment:
Does this work for you? I had to change mine to:
```
"googlemaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!systemjs-googlemaps/googlemaps"
```
to get it to work.

Did you change anything else in your config.js?